packages/sdk/src/lib/dataProtectorCore/processProtectedData.ts

```diff
@@ -38,6 +39,7 @@ export const processProtectedData = async ({
   inputFiles,
   secrets,
   workerpool,
+  useVoucher,
```

Comment:
I would set the default value for `useVoucher` here and revert the change made to `dataProtectorCore/IExecDataProtectorCore.ts`

```suggestion
  useVoucher = false,
```